website/content/docs/k8s/connect/ingress-controllers.mdx

```diff
@@ -0,0 +1,74 @@
+---
+layout: docs
+page_title: Ingress Controller Integrations
+description: Configuring Ingress Controllers With Consul On Kubernetes
+---
+
+# Configuring Ingress Controllers with Consul on Kubernetes
+
+-> This topic requires familiarity with [Ingress Controllers](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/)
+
+-> The following information requires Consul 1.10+, Consul-k8s 0.26+, Consul-helm 0.32+ configured with
+[Transparent Proxy](/docs/connect/transparent-proxy) mode enabled.
+
+This page describes a general approach for integrating Ingress Controllers with Consul on Kubernetes to secure traffic from the Controller
+to the backend services. This allows Consul to transparently manage secure traffic from the ingress point through the entire traffic flow of the service.
+
+A few steps are generally required to enable an Ingress controller to join the mesh and pass traffic through to a service:
+
+* Enable connect-injection via an annotation on the Ingress Controller's deployment: `consul.hashicorp.com/connect-inject` is `true`.
+
+* Using the following annotations on the Ingress controller's deployment, setup exclusion rules for it's ports.
+  * `consul.hashicorp.com/transparent-proxy-exclude-inbound-ports` - Provides the ability to exclude a list of ports for
+inbound traffic that the service exposes from redirection. Typical configurations would require all inbound service ports
+for the controller to be included in this list.
+  * `consul.hashicorp.com/transparent-proxy-exclude-outbound-ports` - Provides the ability to exclude a list of ports for
+outbound traffic that the service exposes from redirection.
+  * `consul.hashicorp.com/transparent-proxy-exclude-outbound-cidrs` - Provides the ability to exclude a list of CIDRs that
```

Comment:
This is not documented on the list of supported annotations. In fact, the list incorrectly documents the availability of a [`consul.hashicorp.com/transparent-proxy-exclude-inbound-cidrs`](https://www.consul.io/docs/k8s/connect#consul-hashicorp-com-transparent-proxy-exclude-inbound-cidrs) annotation, which does not exist. That doc should be updated to reference this annotation for excluding outbound CIDRs.

IDK If you want to lump this in with your PR, or address that discrepancy separately.

Reply:
Fixed good catch!